src/main/scala/org/graphframes/GraphFrame.scala

```diff
@@ -32,76 +32,188 @@ import org.graphframes.pattern._
 
 
 /**
+ * A representation of a graph using [[DataFrame]]s.
+ *
  * Represents a graph with vertices and edges stored as [[DataFrame]]s.
```

Comment:
Remove this paragraph of doc since it is duplicated by stuff below?  And since it's hard to get links to work in the class header Scaladoc.